Settings.c

```diff
@@ -146,19 +153,16 @@ static bool Settings_validateMeters(Settings* this) {
    return anyMeter;
 }
 
-static void Settings_defaultMeters(Settings* this, unsigned int initialCpuCount) {
+static void Settings_defaultMeters(Settings* this, const Machine* host) {
+   unsigned int initialCpuCount = host->activeCPUs;
```

Comment:
Any reason for not using `size_t` here?
```suggestion
   size_t initialCpuCount = host->activeCPUs;
```

Reply:
Sure thing, will do.